pyproject.toml

```diff
@@ -33,6 +34,10 @@ tests = [
   "pytest-mock>=3.14.0"
 ]
 
+parler = [
+  "parler_tts @ git+https://github.com/huggingface/parler-tts.git",
+]
+
```

Comment:
We cannot add this because it breaks the pypi publishing workflow. We need to point users to the repo in order to install it. 

```suggestion
```